tests/python/tir-transform/test_tir_transform_inject_ptx_async_copy.py

```diff
@@ -482,6 +482,12 @@ def simple_compute(
     assert generated_code == expected_cuda_script
 
 
+@pytest.mark.skip(
```

Comment:
I like the documentation, both to indicate what needs to be resolved in order to re-enable functionality, and where the existing discussions are